bundles/org.openhab.binding.emby/src/main/java/org/openhab/binding/emby/internal/protocol/EmbyHTTPUtils.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.emby.internal.protocol;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.io.net.http.HttpUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * HTTP Get and Put reqeust class.
+ *
+ * @author Zachary Christiansen - Initial contribution
+ */
+@NonNullByDefault
+public class EmbyHTTPUtils {
+    private final Logger logger = LoggerFactory.getLogger(EmbyHTTPUtils.class);
+    private int requestTimeout;
+    private String apiKey;
+    private String hostIpPort;
+    private String logTest = "";
+
+    public EmbyHTTPUtils(int requestTimeout, String apiKey, String hostIpPort) {
+        this.requestTimeout = (int) TimeUnit.SECONDS.toMillis(requestTimeout);
+        this.apiKey = apiKey;
+        this.hostIpPort = hostIpPort;
+    }
+
+    public void setRequestTimeout(int requestTimeout) {
+        this.requestTimeout = (int) TimeUnit.SECONDS.toMillis(requestTimeout);
+    }
+
+    /**
+     * Sends a {@link doPost} request with a timeout of 30 seconds
+     *
+     * @param urlAddress the address to send the request
+     * @param payload the json payload to include with the request
+     */
+    private String doPost(String urlAddress, String payload) throws IOException {
+        urlAddress = "http://" + hostIpPort + urlAddress;
+        logger.debug("The String url we want to post is : {}", urlAddress);
+        logger.debug("The payload we want to post is: {}", payload);
```

Comment:
Can this message be replaced by using the debugger or by increasing the framework's log level?